.gitignore

```diff
@@ -8,3 +8,5 @@ node_modules/*
 /compiled
 /secret
 *.log
+.vscode
+.idea
```

Comment:
I think that `.vscode` and `.idea` are unnecessary. Could you remove them?

Reply:
I just added them, because I use both editors and always have to be careful not to commit those IDE specific files. Is it a real problem that we add them here?

Reply:
I understand your opinion, but I think that we should merge only generally code (not only for us)  because Boostnote is open source.

Reply:
We don't think it is unnecessary to add `.idea` and `.vscode` into `.gitignore`. However, if many contributors want it, we'll add it. 😄 
And,  This pull request is "fixed global styles". could you create new pull request for add it.?🙏